### ISSUE TITLE
installation error

### ISSUE THREAD

**Issue by Kushalnahata17** [TridotsTech/Go1-Commerce#2]
Not able to install in Frappe Cloud. throwing error

**Comment by Kushalnahata17** [comment]
Validation Failed: Invalid app release error name

**Comment by mumbaigirl** [comment]
@Kushalnahata17 , I think this can't be installed , if you have installed ERPNext on bench , which is very strange!

 Hopefully the team @TridotsTech will fix the issue. I am also waiting for their reply. Ideally should work with ERPNext v15.

**Comment by clayrisser** [comment]
Can you confirm if it's the same error as https://github.com/TridotsTech/go1-commerce/issues/4?

**Comment by mumbaigirl** [comment]
@clayrisser Yes, did you find any solutions? It seems that the way they have coded it is meant for it to be used as a standalone app. This is a  big dealbreaker. Maybe it was their client's requirements, but to be honest, this doesn't quite fit into the Frappe ecosystem. Otherwise, the app looks decent.

**Comment by clayrisser** [comment]
The only solution would be using it separately and then using something like keycloak to SSO into both platforms.

https://www.keycloak.org
